test/bin/run-tests-docker.sh

```diff
@@ -4,34 +4,80 @@ set -e
 unset CDPATH
 cd "$( dirname "${BASH_SOURCE[0]}" )/../.."
 
-IMAGE_NAME_SCITRAN_CORE=scitran-core
+IMAGE_NAME_SCITRAN_CORE=${IMAGE_NAME_SCITRAN_CORE:-"scitran-core"}
```

Comment:
I think it's cleaner if we don't allow caller to specify the value of this var, and if we include a tag name to avoid collisions. Something like `scitran-core:run-tests`